### PR TITLE
FLA-1165 getting migrated code running

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

https://justice.gov.bc.ca/jira/browse/FLA-1165

Modified the API source a little to get it working in this new repo and copied relevant docker-compose content from jag-file-submission.

I'm able to launch everything with `docker-compose up` and the app appears to run great.

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

docker-compose up -d --build
(then access the application and all appears to run smoothly)

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -12,3 +12,165 @@ services:
...
+    hostname: efiling-reviewer-api
+    build:
+      context: ./src/backend
+      dockerfile: Dockerfile.efiling-reviewer-api
+      args:
+        - SKIP_TESTS=true
+        - MVN_PROFILE=${MVN_PROFILE:-efiling-reviewer}
+        - STARTERS_V=v0.1.8
+    ports:
+      - "8090:8090"
```
Comment (TayGov) [inline docker-compose.yml:30]:
```suggestion
      - "8090:8080"
```

File: src/backend/efiling-reviewer-api/src/main/resources/application.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 server:
-  port: ${SERVER_PORT:8080}
+  port: ${SERVER_PORT:8090}
```
Comment (TayGov) [inline src/backend/efiling-reviewer-api/src/main/resources/application.yml:2]:
```suggestion
  port: ${SERVER_PORT:8080}
```

**Review by TayGov — CHANGES_REQUESTED** [review]
My only suggestion is the port remain default to 8080 as this has implications in openshift. Otherwise this is awesome!

File: docker-compose.yml (hunk 1 above)
Comment (KDNeufeld) [reply docker-compose.yml]: sounds good

**Review by TayGov — APPROVED** [review]

**Pull request merged**